### DISCUSSION TITLE
Zone triggering

### DISCUSSION THREAD

**Discussion by sygys** [blakeblackshear/frigate#3262] [Ideas]
Is it possible to set which corner or side of a motion/bounding box inside a zone will trigger a detection event? I read something that the bottom center of a box inside a zone will trigger. I would like to change this to the top or corners but only on one zone. If this isn't yet possible I would very much like to see this option added in a future update

**Comment by NickM-27** [comment]
This currently isn't possible, and yes it is the bottom center of the bounding box. 

It would help, both in the potential development of this feature as well as offering some suggestions for now, if you could explain the scenario / setup where a zone with the bottom center isn't a good option and another point on the box works better.

**Comment by sygys** [comment]
I would like to let the system detect if we have visitors. most likely they always sit in the chairs in the yellow zone on my picture here. But when sitting there the bounding box of the person will be always lower. This means i need to make the zone bigger to the bottom. but doing so when we stand up from the couch moving left it will trigger the visitor zone. This is only one of the many problems i have encountered with the trigger point being at the bottom.

![Knipsel](https://user-images.githubusercontent.com/73538763/170766149-a9492ed3-2863-477f-887f-96f13a8dc51b.JPG)

If there could be an option in the zones part of the yaml code where you can just say

boundingbox: bottom,left,right,top and maybe topleft, topright bottomleft and bottomright. then you can very easy make more accurate triggers

**Reply by NickM-27** [reply]
Thanks, that is helpful. I think this makes sense, I'll take a look and see how difficult it would be to implement.

**Reply by sygys** [reply]
Thank you very much. This would help allot in making more zones and triggers possible that would else overlap or false trigger

**Reply by Minglarn** [reply]
This would be a SUPERB idea! 
I too have the same problem with bounding box reacting to bottom center objects. 
![image](https://user-images.githubusercontent.com/58695742/170767598-d279ce3e-79cf-4b76-8361-fa4011c9c2b7.png)
In this case both the car on the "street" and the car on my parking is triggered, thus giving me 2 cars found even if only 1 car is new.

**Reply by NickM-27** [reply]
> In this case both the car on the "street" and the car on my parking is triggered, thus giving me 2 cars found even if only 1 car is new.

I'm not sure you're talking about the same thing as the original issue, this is only referring to objects being considered in a zone not how objects are detected using regions or anything like that. But I may be misunderstanding what you were referring to from the screenshot.

**Reply by NickM-27** [reply]
[Draft PR is up](https://github.com/blakeblackshear/frigate/pull/3263), probably won't be making into the 0.11 release but we'll see

**Reply by sygys** [reply]
Ok thanks this is awesome. Nice to see this project is so we'll supported. I love frigate.
